tiflash/use-tiflash.md

```diff
@@ -79,6 +79,65 @@ In the result of above statement:
 * `AVAILABLE` indicates whether the TiFlash replicas of this table is available or not. `1` means available and `0` means unavailable. Once the replicas become available, this status does not change. If you use DDL statements to modify the number of replicas, the replication status will be recalculated.
 * `PROGRESS` means the progress of the replication. The value is between `0.0` and `1.0`. `1` means at least one replica is replicated.
 
+## Create TiFlash replicas for databases
+
+Similar with creating TiFlash replicas for tables, you can send a DDL statement to TiDB through a MySQL client to create a TiFlash replica for all tables in a specific database:
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER DATABASE db_name SET TIFLASH REPLICA count
+```
+
+In this statement, `count` indicates the number of replicas. When the value is `0`, the replica is deleted.
```

Comment:
Should we use `the replicas will be deleted`? Since `ALTER DATABASE` involves replica for many tables